tests/acceptance/organization-test.js

```diff
@@ -0,0 +1,93 @@
+import { describe, it, beforeEach, afterEach } from 'mocha';
+import _ from 'lodash';
+import { expect } from 'chai';
+import startApp from '../helpers/start-app';
+import destroyApp from '../helpers/destroy-app';
+
+describe('Acceptance | organization', () => {
+  let application;
+
+  beforeEach(() => {
+    application = startApp();
+  });
+
+  afterEach(() => {
+    destroyApp(application);
+  });
+
+  describe('Organization page', () => {
+    beforeEach(function() {
+      const memberCount = 3,
+            repositoryCount = 2,
+            members = server.createList('user', memberCount),
+            organization = server.create('organization', {
+              memberIds: _.map(members, (u) => u.id)
```

Comment:
According to http://www.ember-cli-mirage.com/docs/v0.3.x/models/#associations it should work (if the factories call the same methods as described there).